Azure-Samples/chat-with-your-data-solution-accelerator
# fix: Focus of screen shifts whenever we type question in search box

## Purpose
Deploy CWYD
Ask few questions ex. 2/3 
When you have enough chat history like you can see the vertical scroll bar to scroll through the history. 
Try typing question in search box and observe the behavior of focus on screen 
It shifts up and down continuously. 
I have attached video for reference in attachments.   

## Does this introduce a breaking change?





## How to Test

- Upload PDF file and ask some question related to that PDF
- you will get the response which is including the reference part with enable arrow ">"
- click on it to enable the reference and keep it enabled and scroll to the top 
- now ask some more question and check the screen is going up and down 



*  Get the code
```
git clone [[repo-address](https://github.com/Azure-Samples/chat-with-your-data-solution-accelerator)]

git checkout [psl-bug-7550]
```

please refer the below screenshot 
![image](https://github.com/user-attachments/assets/da692962-2de4-4ddb-8002-e5507c8eddb0)
